nvml-wrapper/src/device.rs

```diff
@@ -2672,6 +2672,58 @@ impl<'nvml> Device<'nvml> {
         }
     }
 
+    /**
+     Get GPU instance placements. A placement is a given location of a GPU in a device.
+
+    # Errors
+
+    * `Uninitialized`, if the library has not been successfully initialized
+    * `InvalidArg`, if this `Device` is invalid
+    * `NotSupported`, if this `Device` does not support this feature
+    * `GpuLost`, if this `Device` has fallen off the bus or is otherwise inaccessible
+    * `Unknown`, on any unexpected error
+
+    # Platform Support
+
+    Only supports Linux.
+
+    # Device Support
+
+    Supports Ampere and newer fully supported devices.
+    */
+    #[cfg(target_os = "linux")]
+    #[doc(alias = "nvmlDeviceGetGpuInstancePossiblePlacements_v2")]
+    pub fn possible_placements(
+        &self,
+        profile: u32,
+    ) -> Result<Vec<GpuInstancePlacement>, NvmlError> {
+        let sym = nvml_sym(
+            self.nvml
+                .lib
+                .nvmlDeviceGetGpuInstancePossiblePlacements_v2
+                .as_ref(),
+        )?;
+
+        unsafe {
+            let mut count: c_uint = 0;
+            nvml_try(sym(self.device, profile, ptr::null_mut(), &mut count))?;
+            let mut placements: Vec<nvmlGpuInstancePlacement_t> =
+                Vec::with_capacity(count as usize);
+
+            nvml_try(sym(
+                self.device,
+                profile,
+                placements.as_mut_ptr(),
+                &mut count,
+            ))?;
+
+            Ok(placements
+                .into_iter()
+                .map(|p| GpuInstancePlacement::from(p))
```

Comment:
You've got a clippy warning here
```suggestion
                .map(GpuInstancePlacement::from)
```

(I realize there's a lot of them in the codebase, but let's not add more)